### DISCUSSION TITLE
Model Bounties

### DISCUSSION THREAD

**Discussion by JustMaier** [civitai/civitai#4] [Ideas]
Request models etc. that you'd like to see and attach value to it that someone can earn by submitting a model and being awarded the bounty by the requester.

Bounties and their results would be public by default, but maybe in the future the bounty poster could pay extra to keep the bounty result private.

### Video Overview
[Watch the video](https://dl.dropboxusercontent.com/s/byevgqrp3mj3rzd/2022-12-14%2013-38.mp4?dl=0)  (3:29 min)

### Bounties Listing
![Bounties listing mockup](https://user-images.githubusercontent.com/607609/207710485-e2e327fc-d5aa-409a-9c8d-00b33a5c2cfd.png)

### Bounty Page
![Bounty page mockup](https://user-images.githubusercontent.com/607609/207710542-f6f046d4-f067-4119-8450-3a9c0e9a86ef.png)

### Bounty Creation
![Bounty Creation mockup](https://user-images.githubusercontent.com/607609/207710630-07441d1c-0ada-4f91-934d-849b545d10ba.png)

**Comment by Sylennus** [comment]
I stumbled here trying to find a suggestion page on CivitAI, hopefully this is the appropriate place to post this :

Unless I'm mistaken, right now only the author of the bounty gets access to the awarded file(s). My suggestion would be to allow any user to award additional buzz to any entry as long as the bounty is open. It would allow participants/creators to earn more buzz and maybe spark additional interest in fulfilling bounties. Also an option to automatically release an entry publicly (with its creator as the author) if it crosses a buzz threshold ? I can't say I'm too happy about content getting exchanged between only 2 users and this would partly alleviate that.
